src/Bicep.Core/CoreResources.resx

```diff
@@ -286,4 +286,15 @@
   <data name="MaxNumberResourcesRuleMessageFormat" xml:space="preserve">
     <value>Too many resources. Number of resources is limited to {0}.</value>
   </data>
+  <data name="PreferBarePropertyNamesDeclarationFixTitle" xml:space="preserve">
+    <value>Replace quoted property name with {0}.</value>
+    <comment>{0} is a programmatic language expression</comment>
+  </data>
+  <data name="PreferBarePropertyNamesDereferenceFixTitle" xml:space="preserve">
+    <value>Replace array syntax with {0}.</value>

```

Comment:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/6913354/171232531-fa746d7c-4d0d-41d7-820c-d494968573d4.png">
```suggestion
    <value>Replace array syntax with '{0}'</value>
```
@ucheNkadiCode Good?  With change with be:
```
Replace array syntax with '.myProp'
```

Reply:
Maybe 
```
Replace array syntax with '<expr>.myProp'
```
?

Reply:
I like "Replace array syntax with 'obj.myProp', more clear

Reply:
Updated in https://github.com/Azure/bicep/pull/6584/commits/ec1247ddd5f2a4aaf7d18e7ad1493dcc4e353e91:

![quickFixTitles](https://user-images.githubusercontent.com/705500/171301392-1220c6e5-6aef-486d-b0ca-4b627a23c0b1.gif)